volontulo/views/organizations.py

```diff
@@ -44,8 +45,24 @@ def get(request):
         )
 
     @staticmethod
+    @login_required
     def post(request):
         u"""Method responsible for saving new organization."""
+        if not (
+                request.POST.get('name') and
+                request.POST.get('address') and
+                request.POST.get('description')
+        ):
+            yield_message_error(
+                request,
+                u"Należy wypełnić wszystkie pola formularza."
+            )
+            return render(
+                request,
+                "organizations/organization_form.html",
+                {'organization': Organization()}
```

Comment:
This should be fixed within templates. So I see another problem with my test - it should also check if returned page contains filled fields, even if error occured.